src/translation/raw_data/campaigns_site_static_texts.json

```diff
@@ -0,0 +1,349 @@
+{
```

Comment:
this should be in the frontend and there should be an endpoint which can be called to translate this in the backend and return it.